vnext/Common/unicode.cpp

```diff
@@ -13,14 +13,12 @@
 #include <exception>
 #include <string>
 
-namespace facebook {
-namespace react {
-namespace unicode {
+namespace Microsoft::Common::Unicode {
```

Comment:
>Common [](start = 21, length = 6)

So, I don't see comments from previous CR, but are we set on calling this "Common"? If we have "Shared" and "Common" it could be confusing.

Reply:
I am not 100% sure about the name "Common" either. Personally, I view JSI and React as two separate modules, with React depending on JSI and not the other way around. While Shared.vcxitems contains code that is shared between the React projects, Common.vcxproj is supposed to contain code that is shared between React and JSI projects. Hence, I'd rather not put code in Common into Microsoft::React or Microsoft::JSI. Julio and I chatted and the best name we could come up with is Common. I think we should rename Shared.vcxitems into something like React.Windows.Shared.vcxitem, but that should happen in a separate PR.

---
In reply to: [333779845](https://github.com/microsoft/react-native-windows/pull/3390#discussion_r333779845) [](ancestors = 333779845)

Reply:
Why not name it something like "Javascript" or "Js", or something related to that? "Common" is overly generic and doesn't convey that this is related to javascript-ish things in any way. Please reconsider. Ask around as well for other people's opinions.

---
In reply to: [333783007](https://github.com/microsoft/react-native-windows/pull/3390#discussion_r333783007) [](ancestors = 333783007,333779845)